src/tester.rs

```diff
@@ -431,6 +431,24 @@ fn check_names<'a>(cmd_pairs: &[(String, Command)], tests: &HashMap<String, Test
     }
 }
 
+fn write_ignored(test_name: &str, message: &str, single_threaded : bool) {
```

Comment:
Can we move this function to just after (or before) `write_with_colour`?

Reply:
Fixed: 86ff69501c2fb06148d612175836666cdebae040

Reply:
I think you might have referenced the wrong commit. I think you meant https://github.com/softdevteam/lang_tester/pull/36/commits/729678dd3cbcb414eaf6a49dd655a958d731fa15.

Reply:
Well done you passed the test.